features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -134,41 +135,60 @@ class ApiConsole extends React.Component {
             })
             .then((swaggerResponse) => {
                 swagger = swaggerResponse.obj;
-                return this.apiClient.getSubscriptions(apiID);
+                if (user != null) {
```

Comment:
Can't we just check `if (user)` instead of checking `user != null`?

Reply:
fixed in 413eb4748a5e0a6ad71d4e9a19db6da8c62c6832